firstvoices/firstvoices/settings.py

```diff
@@ -120,6 +120,55 @@
 # LOGGERS
 ELASTICSEARCH_LOGGER = "elasticsearch"
 
+# Based on the default settings
+# ref: https://docs.djangoproject.com/en/5.1/ref/logging/#default-logging-configuration
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "filters": {
+        "debug_info_log_level_filter": {
+            "()": "firstvoices.logging_utils.DebugInfoLogLevelFilter"
+        },
+    },
+    "formatters": {
+        "standard": {
+            "format": "{levelname}-{asctime}:{pathname} -- {message}",
+            "style": "{",
+        },
+        "detailed": {
+            "format": "{levelname}-{asctime}:{pathname} {process:d} {thread:d} -- {message}",
+            "style": "{",
+        },
+    },
+    "handlers": {
+        "console": {
+            "level": "DEBUG",  # DEBUG + INFO
```

Comment:
Since this filters all logs above DEBUG (i.e. DEBUG, INFO, WARNING, ERROR, CRITICAL), a custom filter is added to only filter the DEBUG and INFO logs.